LessonSix/src/test/java/TestArray.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+
+
+
+    @RunWith(Parameterized.class)
+    public class TestArray {
+        @Parameterized.Parameters
+        public static Collection<Object[]> data() {
+            return Arrays.asList(new Object[][]{
+                    {new int[]{1, 6, 5, 4, 2, 3, 3}, new int[]{2, 3, 3}},
+                    {new int[]{4, 3, 7, 5, 4, 4, 3, 2}, new int[]{3, 2}}
+            });
+        }
+
+        private int[] in;
+        private int[] out;
+
+        public TestArray(int[] in, int[] out) {
+            this.in = in;
+            this.out = out;
+        }
+
+        private Array arr;
+
+        @Before
+        public void startTest() {
+            arr = new Array();
+        }
+
+        @Test
+        public void testAfterLast4() {
+            Assert.assertArrayEquals(out, Array.AfterLast4(in));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.